src/FTPClient.cpp

```diff
@@ -35,7 +35,7 @@ TFTPClient::TFTPClient()
 		return;
 	}
 	
-	// 受信スレッド生成
+	// Create receive thread
```

Comment:
receiving thread

Reply:
Then it should be rather "Create receiver thread"